lib/lte_link_control/Kconfig

```diff
@@ -344,6 +344,11 @@ config LTE_LC_TRACE
 	  what has happened and has no associated payload.
 	  It is intended to be used for debugging purposes.
 
+config LTE_LC_WORKQUEUE_STACK_SIZE
+	int "Stack size for the library work queue"
+	default 1280 if LOG_MODE_IMMEDIATE
+	default 896
```

Comment:
ok to bump the size when `LOG_MODE_IMMEDIATE` but I wouldn't fine tune the default too much or risk having to increase it as soon as something gets added to workqueue execution context.

Reply:
Naturally, if something is added to the work queue, the size has to be always re-evaluated. But I'll keep the default size for now.